firefox/manifest.json

```diff
@@ -80,6 +80,7 @@
 		"https://api.tumblr.com/v2/blog/*/posts",
 		"https://xkcd.com/*/info.0.json",
 		"https://api.steampowered.com/ISteamRemoteStorage/GetPublishedFileDetails/*",
-		"https://content.googleapis.com/drive/v3/*"
+		"https://content.googleapis.com/drive/v3/*",
+		"https://www.googleapis.com/oauth2/v3/*"
```

Comment:
Can these be moved to optional permissions now? Seems that F[irefox ≥ 55 support that](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/optional_permissions#Browser_compatibility).

Reply:
Theoretically; see the ff-perms branch in my fork. Though I always seem to get `Error: May only request permissions from a user input handler`. I assume it doesn't handle asynchronicity (which is going to be annoying), but calling `chrome.runtime.sendMessage` directly from an event listener doesn't fix it either...

Reply:
Since adding optional permissions in Firefox seems like a non-trivial task, how about we get this in first and then handle the permissions in a separate PR?